### PR TITLE
Remove unused return value from `_check()` method

### PR DESCRIPTION
**Pull request by 0xjei**
## Description

This PR removes the unused return value from `_check()` method from `Excubia` abstract contract. See this [comment for more](https://github.com/privacy-scaling-explorations/zk-kit.solidity/pull/20#discussion_r1664169140).

## Related Issue(s)
none

## Other information
none

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] My changes generate no new warnings
-   [x] I have run `yarn format` and `yarn compile` without getting any errors
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/main.yml
Comment (cedoor) [inline .github/workflows/main.yml:1]: These changes should not be part of this PR. I think this branch is synced with the main one and `feat/porting-maci-gatekeepers` is not. You should solve it by running `git rebase main` on `feat/porting-maci-gatekeepers`.

**Review by cedoor — APPROVED** [review]
LGTM! Just one comment (not related to your code).

File: .github/workflows/main.yml
Comment (0xjei) [reply .github/workflows/main.yml:1]: tysm - done!
